### PR TITLE
[MM-39057] -  Remove Tips & Next Steps page

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR removes Tips and Next Steps page and all related code from the onboarding

#### Ticket Link
[MM-39057](https://mattermost.atlassian.net/browse/MM-39057)

#### Release Note
```release-note
NONE

```

### CODE REVIEW

**Comment by neallred** [comment]
Huge fan of the over 2K LOC removed :grin:

File: components/sidebar/sidebar_next_steps/sidebar_next_steps.scss [hunk 1]
```diff
@@ -1,8 +1,4 @@
 .SidebarNextSteps {
-    &.tips {
-        height: 70px;
-    }
-
     position: relative;
     width: 100%;
     height: 82px;
```
Comment (neallred) [inline components/sidebar/sidebar_next_steps/sidebar_next_steps.scss:4]: Looks like the `.close-icon` styles on lines 72-94 can also be removed.

File: components/sidebar/sidebar_next_steps/sidebar_next_steps.scss [hunk 2]
```diff
@@ -1,8 +1,4 @@
 .SidebarNextSteps {
```
Comment (neallred) [inline components/sidebar/sidebar_next_steps/sidebar_next_steps.scss:1]: Looks like there's a number of `.NextStepsView__nextStepsCards` styles that can all be removed.

**Comment by neallred** [comment]
LGTM besides the additional styles we can remove.

**Review by pablovelezvidal — APPROVED** [review]
LGTM! :shipit:

**Comment by marianunez** [comment]
@neallred gentle ping for this to try to make it to the next cloud release

**Review by neallred — APPROVED** [review]
Non blocking suggestion to remove unused classes above.

**Review by stevemudie — APPROVED** [review]
@AGMETEOR No Tips & Next Steps to be seen as expected.

**Comment by marianunez** [comment]
@AGMETEOR is this waiting on something else ? I noticed your `Do not Merge` tag

**Comment by AGMETEOR** [comment]
> @AGMETEOR is this waiting on something else ? I noticed your `Do not Merge` tag

I wanted to merge https://github.com/mattermost/mattermost-webapp/pull/9188 first

**Pull request merged**